.github/workflows/mac.yml

```diff
@@ -11,9 +11,6 @@ on:
         default: ""
         required: false
         type: string
-      wpt-layout:
```

Comment:
We shouldn't remove this parameter. They will be updated as part of a different issue.